crates/hotshot/task-impls/src/helpers.rs

```diff
@@ -1428,3 +1431,22 @@ pub async fn broadcast_view_change<TYPES: NodeType>(
     )
     .await
 }
+
+pub fn derive_lc_state_digest(
+    lc_state: &LightClientState,
+    next_stake_state: &StakeTableState,
+    auth_root: &U256, // TODO(Chengyu): replace with actual types
```

Comment:
should be bytes32? since it's the output of keccak256.

Reply:
yes, changed in [b21f5c8](https://github.com/EspressoSystems/espresso-network/pull/3463/commits/b21f5c8ddef7a85566ac7c31c7090a3de91ee93b)